### PR TITLE
Upload Image

### PR DESCRIPTION
**Pull request by diegorosolino**
### Ticket
[Update Mobile Topics Views to newest designs](https://www.notion.so/usedispatch/A-user-is-able-to-upload-a-picture-as-a-post-or-reply-on-a-forum-topic-634c54e82bb44884918a4591fcc788b6)

### Description
This pr adds the uploaded image in the replay or post

### Screenshots
Desktop:

https://user-images.githubusercontent.com/57630050/199611550-2c10a29e-13ca-4f8d-be0f-02ca8aba8d29.mp4

Mobile:

https://user-images.githubusercontent.com/57630050/199611591-fdc551d1-f734-48d5-b41e-20d4c53af41d.mp4

### CODE REVIEW

File: src/components/forums/topic/UploadTopicImage.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import * as _ from "lodash";
```
Comment (tj-dispatch) [inline src/components/forums/topic/UploadTopicImage.tsx:1]: Does this need to be in here? Do you use it anywhere?

**Review by tj-dispatch — APPROVED** [review]
LGTM -- just check that lodash import. If we're not using it, let's get rid of it. If we are using it, let's not import it all and instead only the methods we need. LMK if you have any questions / if there's a reason to keep it in there and you can land it after!

**Comment by zfaizal2** [comment]
tried making an upload on this preview link with your changes, but getting some console errors here
https://forum-staging-giumu6leg-usedispatch.vercel.app/demos/topic/41
image

**Comment by tj-dispatch** [comment]
Is that url supposed to be http and not https?

**Pull request merged**